### ISSUE TITLE
Docker Hub Image ekofr/pihole-exporter:arm fails to run on Pi 3B+ (v7 32bit)

### ISSUE THREAD

**Issue by logopk** [eko/pihole-exporter#57]
running the image official image ekofr/pihole-exporter:arm results in 
`pihole-exporter    | standard_init_linux.go:211: exec user process caused "exec format error"
`
**Steps for Reproduction**

run the image

**Expected behavior**:

Container starts

**Actual behavior**:

Startup fails with 
`pihole-exporter    | standard_init_linux.go:211: exec user process caused "exec format error"
`

**Platforms**:
Linux raspberrypi 4.19.118-v7+ #1311 SMP Mon Apr 27 14:21:24 BST 2020 armv7l GNU/Linux

Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a6621
 Built:             Mon Jun 22 15:53:41 2020
 OS/Arch:           linux/arm
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a6621
  Built:            Mon Jun 22 15:47:34 2020
  OS/Arch:          linux/arm
  Experimental:     true
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

**Versions**:

Which versions are you running?
ekofr/pihole-exporter:arm

**Comment by eko** [comment]
Hi @logopk,

Could you please try with the `ekofr/pihole-exporter:0.0.9` tag?

Link here: https://hub.docker.com/layers/ekofr/pihole-exporter/0.0.9/images/sha256-9c32c4b09a3d271501896bcd938f0a54bb6e4c9bd6b44e42e77c44814ac3ea8d?context=explore

This image supports multiple architectures and it should work.

Thank you

**Comment by logopk** [comment]
Hi @eko ,
:0.0.9 works fine. That's what I'm using. I was only curious, as I got notified (by diun https://github.com/crazy-max/diun) that there was an update on :latest.

Still interesting why :0.0.9 as multi arch works and :arm does not.

Thanks.

**Comment by derekmurawsky** [comment]
I am also having this error. No problems using 0.0.9.

**Comment by 91ajames** [comment]
Same here, 0.0.9 worked for me.

**Comment by TrikkStar** [comment]
Chiming in to say that I needed to specify 0.0.9 as well (Pi 3)

**Comment by jariz** [comment]
Can we maybe get a 0.0.10 release? I need #56 because my admin is running on a different port.

Building the image on my pi results in the following error:
```
ERROR: unsatisfiable constraints:
  upx (missing):
    required by: world[upx]
```
which strangely doesn't happen on my macbook.

**Comment by eko** [comment]
Hi @jariz,

Indeed the upx package does not seems to be still available for arm architecture on [Alpine 3.12](https://pkgs.alpinelinux.org/packages?name=upx&branch=v3.12) version but it was on [Alpine 3.11](https://pkgs.alpinelinux.org/packages?name=upx&branch=v3.11).

I've forced the Alpine 3.11 version for a quick fix but I will have a look on how I can install it without the alpine apk package manager.

Also, I just tagged & built a [`ekofr/pihole-exporter:0.0.10`](https://hub.docker.com/layers/ekofr/pihole-exporter/0.0.10/images/sha256-52ad74a178f47f04ff69c5eead6289948482e74b8617e3ee342bd1c0c716bef3?context=repo) docker image which is available on the docker registry right now.

Thank you

**Comment by jariz** [comment]
Thanks @eko! 
Works great now!
